cli/README.md

```diff
@@ -0,0 +1,42 @@
+# generator-mern [![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url]
+> A Yeoman generator to scaffold a MERN app
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+```
+
+Then link the generator-mern because is not available as global npm module. On the command line, from the root of your generator project (in the /cli folder), type:
+
+```bash
+npm link
+```
+
+That will install your project dependencies and symlink a global module to your local file.
+
+## Generators
+
+### Component
+
+Generates a component in `app/components`.
+
+Example:
+
+```bash
+yo mern:component Sidebar
```

Comment:
Hi @matiasbeckerle ... I was thinking while sleeping and I think the interesting thing that we can do here is to have 3 different commands (or 3 differents optional parameters - I leave that up to you) to create the 3 different type of components you can have in React-Redux project then.
For example: (I'll not create the templates but simple examples)
1- `React Class Component` (the one you've created)
```
import React, {PropTypes} from 'react';

class ReactClassComponent extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div>
                ---Autogenerated <%= name %>---
            </div>
        );
    }
}

ReactClassComponent.defaultProps = {

};

ReactClassComponent.propTypes = {

};

export default ReactClassComponent
```

2- `React Stateless Component` 
```
import React, {PropTypes} from 'react';

const ReactStatelessComponent = (props) => {
    return (
        <div></div>
    );
};

ReactStatelessComponent.defaultProps = {
    
};

ReactStatelessComponent.propTypes = {
    
};

export default ReactStatelessComponent;
```

3- `React-Redux Container Component`
```
import React, {PropTypes} from 'react';
import {connect} from 'react-redux';
import {bindActionCreators} from 'redux';

class ReactReduxContainerComponent extends React.Component {
    constructor(props, context) {
        super(props, context);
    }

    render() {
        return (
            <div></div>
        );
    }
}

ReactReduxContainerComponent.defaultProps = {
   
};

ReactReduxContainerComponent.propTypes = {
   
};

function mapStatesToProps(state, ownProps) {
    return {
        state: state
    };
}

function mapDispatchToProps(dispatch) {
    return {
        actions: bindActionCreators(actions, dispatch)
    }
}

export default connect(mapStatesToProps, mapDispatchToProps)(ReactReduxContainerComponent);
```

Do you like the idea @matiasbeckerle ?